### ISSUE TITLE
Forced Subtitles being reset

### ISSUE THREAD

**Issue by rg9400** [RemiRigal/Plex-Auto-Languages#45]
I noticed that when I grabbed the Boys, with PAL set to work on new files and episodes, it seems to have reset the forced subtitles and turned them off. I believe Plex sets Forced Subs on by default, so this seems like it is happening due to PAL's behavior of setting default tracks for new items.

**Comment by RemiRigal** [comment]
You are absolutely correct ! As of today forced subtitles are treated like any other subtitles, which mean that if you have the following tracks:
```plaintext
Subtitles for episode 1:
  - English
  - English (Forced)
Subtitles for episode 2:
  - English
Subtitles for episode 3:
  - English
  - English (Forced)
```
When playing episode 2, PAL sees that you have not selected any subtitles track and therefore unselects the forced track for episode 3, which is undesired of course.

There is two ways to tackle this:
- Always selects a forced subtitles track when available for the selected audio track language (this is what is implemented in #17)
- Never unselect a forced subtitles track.

I've not merged #17 because I don't know if this new behavior will suit most people. Any opinion ?

**Comment by rg9400** [comment]
I feel the second option is the one that makes sense logically. But the first option would end up with the same results in my usecase, and so I am fine with that. Just not sure how it could impact some other scenarios.

But to me, not selecting a subtitle is the equivalent of doing nothing. So nothing should happen when Plex loads the default sub track for the new episode. On the other hand, if I purposefully go in and turn off forced subs for episode 1, this is an actual action, so this means that I am asking for all future forced subs to be turned off as well. I am not sure why someone would do this, but I believe this wouldn't work in your proposed solution.

**Comment by RemiRigal** [comment]
Hi @rg9400, sorry for the delay and thank you for your feedback.

FYI, the latest version will no longer unselects forced subtitles and will look for forced subtitles matching the current audio language if none is selected.
